Week8/MAKEME.md

```diff
@@ -11,24 +18,160 @@ Give feedback on the SPA (Github API) and git branching homework of one of you f
 
 :point_up:
 
-## Step 3: Pair programming promises challenge 
+## Step 3: Read
+
+>Read:
+- JavaScript : [Closures](http://conceptf1.blogspot.nl/2013/11/javascript-closures.html)
+- Everything you wanted to know about [JavaScript scope](https://toddmotto.com/everything-you-wanted-to-know-about-javascript-scope/)
+- JavaScript [Scoping and Hoisting](http://www.adequatelygood.com/JavaScript-Scoping-and-Hoisting.html)
+- 5 JavaScript [“Bad” Parts That Are Fixed In ES6](https://medium.freecodecamp.com/5-javascript-bad-parts-that-are-fixed-in-es6-c7c45d44fd81)
+
+- More about [closures](https://www.reddit.com/r/learnjavascript/comments/1v6n8p/closure_explain_likei_am_in_high_school/?st=ixsp0mbe&sh=5526d150)
+- A VERY popular [StackOverflow article](http://stackoverflow.com/questions/111102/how-do-javascript-closures-work)
+
+## Step 4: Scope and Closures
+
+> Let's continue to learn a little more about scope and Closures. 
+
+
+2. What will be the output of the following code - and more importantly - WHY? 
+
+```js
+for (var i = 0; i < 3; i++) {
+      setTimeout(function() { alert(i); }, 1000 + i);
+}
+```
+
+
+3. Write a function that would allow you to do this:
+```js
+let addSix = createBase(6);
+addSix(10); // returns 16
+addSix(21); // returns 27
+```
+
+4. You will need to create an HTML document out of the below snippet to run the below code. A hint - the code is syntactically correct but doesn't do what you would expect. Can you see why and fix it?
```

Comment:
Because we changed `var` by `let` in the for loop the expected problem no longer occurs!